### PR TITLE
Add a map to guide contributors to Enarx components that match their interests

### PR DESCRIPTION
**Pull request by connorkuehl**
I recommend cloning this branch and opening the website locally. The graph's color scheme was created with our current website's black background in mind.

Signed-off-by: Connor Kuehl <ckuehl@redhat.com>

### CODE REVIEW

**Comment by connorkuehl** [comment]
I gave this a test run with someone who is not familiar with Enarx or our repo organization and their comments were basically that it is opaque and hard to follow...

Does anyone have any ideas for alternatives?

**Review by haraldh — APPROVED** [review]
better than nothing of this sort
